composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3|~3.0",
+        "symfony/symfony": "~2.3",
```

Comment:
I think I know what makes you say that : nothing in the SonataPageBundle code is incompatible with symfony 3.0, only its dependencies are incompatible… but do we really know that, since probably nobody has ever installed SonataPageBundle along with sf 3.0 because of this incompatibility?